### PR TITLE
Updates Pivotal Tracker API gem

### PR DESCRIPTION
**Pull request by adamof**
The `pivotal-tracker` gem was abandoned, so I've changed to the `tracker_api` gem that is maintained and works with the API v5 of Pivotal Tracker.

### CODE REVIEW

File: lib/git-whistles/pull_request/pivotal.rb [hunk 1]
```diff
@@ -28,14 +28,16 @@ def tracker_related_params(team)
...
           begin
-            story, project = PivotalTracker::Project.all.find do |project|
+            story, project = client.projects.find do |project|
               log.info '.¬'
-              story = project.stories.find(story_id) and break story, project
+              story = project.story(story_id) and break story, project
             end
             log.info '.'
-          rescue RestClient::Unauthorized
+          rescue StandardError => e
```
Comment (pedrocunha) [inline lib/git-whistles/pull_request/pivotal.rb]: What is the reason behind this change? Can't we rescue instead an additional rest client error code?

Comment (adamof) [reply lib/git-whistles/pull_request/pivotal.rb]: The new gem doesn't use `RestClient` :/
I don't like this solution either, but this is what the new gem provides.

Comment (pedrocunha) [reply lib/git-whistles/pull_request/pivotal.rb]: Right, why not being explicit on rescuing `ClientError` then?

Comment (adamof) [reply lib/git-whistles/pull_request/pivotal.rb]: hm, I must've tried this. Let me have a look and I'll tell you why not :)
